src/components/Avatar/AvatarMenu.js

```diff
@@ -36,10 +40,19 @@ class AvatarMenu extends Component {
     }
 
     handleAvatarClick = () => {
-        this.setState({ open: !this.state.open });
+        const open = !this.state.open;
```

Comment:
Just for clarity I think this would give a clearer impression of what it's doing if it was named `shouldOpen` or something similar. `open = !open` is a tad confusing.

Reply:
You are right, I'm going to fix that